golem/managers/proposal/plugins/negotiating/mid_agreement_payments.py

```diff
@@ -0,0 +1,96 @@
+import logging
+from datetime import timedelta
+from typing import Optional
+
+from golem.managers.base import ProposalNegotiator, RejectProposal
+from golem.resources import DemandData, ProposalData
+
+logger = logging.getLogger(__name__)
+
+DEFAULT_MIN_DEBIT_NOTE_INTERVAL = timedelta(minutes=1)
+DEFAULT_REQUESTED_DEBIT_NOTE_INTERVAL = timedelta(minutes=10)
+DEFAULT_MIN_PAYMENT_TIMEOUT = timedelta(minutes=2)
+DEFAULT_REQUESTED_PAYMENT_TIMEOUT = timedelta(hours=24)
+
+DEFAULT_MIN_ADJUSTMENT = 1
+DEFAULT_ADJUSTMENT_FACTOR = 3
```

Comment:
As in general I get the idea of adjustment steps, and its `int` nature, I would expect, that `factor` is definitely a float and maybe na optional normalized float. Instead of `3` I would expect to have `0.3` more.